.husky/pre-commit

```diff
@@ -5,5 +5,10 @@ if git diff --cached --name-only --diff-filter=ACM | grep --max-count=1 -E "^des
 fi
 
 if git diff --cached --name-only --diff-filter=ACM | grep -E --max-count=1 '\.scala$|\.sbt$' > /dev/null; then
-  ./sbtwrapper formatStagedScalaFiles
-fi
\ No newline at end of file
+  # use real sbt if possible: https://github.com/dwijnand/sbt-extras/issues/377
+  if hash sbt 2>/dev/null; then
```

Comment:
can we move it to a separate script file e.g. `formatStagedBackendFiles.sh`? To be consistent with the frontend approach done above

Reply:
done